docs/sphinx/dev_guide/release_process.rst

```diff
@@ -0,0 +1,137 @@
+.. ##
+.. ## Copyright (c) 2016-21, Lawrence Livermore National Security, LLC
+.. ## and RAJA project contributors. See the RAJA/COPYRIGHT file
+.. ## for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+.. ##
+
+.. _release-label:
+
+*******************************************
+RAJA Release Process
+*******************************************
+
+The RAJA release process typically involves the following sequence of steps:
+
+  #. Identify all work (features in development, outstanding PRs, etc.) to be 
+     to be included in the release.
+  #. Merge all PRs containing work to be included in the release into the 
+     develop branch.
+  #. Make a :ref:`rcbranch-label` from the develop branch. Finalize the 
+     release by completing remaining release tasks on that branch.
```

Comment:
I'm not sure what's in the remaining release tasks, but would that include making any necessary changes to like the version numbers for the cmake env variables (RAJA_VERSION_****), release_notes, and the docs/conf.py file, etc.?

Reply:
Oh nvm, I see this is all mentioned below...

Reply:
(I assume that's the better place to put it - so I'll resolve this..)